subscription_platform/dashboards/monitor_saasboard__active_subscriptions.dashboard.lookml

```diff
@@ -0,0 +1,950 @@
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    listens_to_filters: [Service ID, Active Date, Plan Interval]
+    field: daily_active_logical_subscriptions.subscription__plan_summary
+  - name: Region
+    title: Region
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    listens_to_filters: [Service ID, Active Date]
+    field: countries.region_name
+  - name: Country
+    title: Country
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    listens_to_filters: [Region, Service ID, Active Date]
+    field: countries.name
+  - name: Has Fraudulent Charges (Yes / No)
+    title: Has Fraudulent Charges (Yes / No)
+    type: field_filter
+    default_value: 'No'
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: dropdown_menu
+      display: overflow
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    listens_to_filters: []
+    field: current_subscription_state.has_fraudulent_charges
+  - name: Has Refunds (Yes / No)
+    title: Has Refunds (Yes / No)
+    type: field_filter
+    default_value: 'No'
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: dropdown_menu
+      display: overflow
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    listens_to_filters: []
+    field: current_subscription_state.has_refunds
+  - name: Service ID
+    title: Service ID
+    type: field_filter
+    default_value: Monitor
+    allow_multiple_values: true
+    required: true
+    ui_config:
+      type: button_toggles
+      display: overflow
+      options:
+      - Monitor
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    listens_to_filters: []
+    field: subscription_services.id
+  elements:
+  - name: Navbar
+    type: text
+    title_text: ''
+    subtitle_text: ''
+    body_text: |
+      <div style="border-radius: 5px; padding: 5px 10px; background: #412399; height: 60px;">
+
+      <nav style="font-size: 20px;">
+
+        <a style="color: #efefef; padding: 5px 25px; float: left; line-height: 40px; border: 1px solid white; font-weight: bold; text-decoration: underline;">
+          Active Subscriptions
+        </a>
+
+        <a href="https://mozilla.cloud.looker.com/dashboards/subscription_platform::monitor_saasboard__subscriptions_growth" style="color: #efefef; padding: 5px 25px; float: left; line-height: 40px;">
+          Subscriptions Growth
+        </a>
```

Comment:
I changed the abbreviated nav links "Active Subs" and "Subs Growth" to be spelled out, figuring we have the room after removing the Looker logo image, but if you prefer them abbreviated I can change them back.

Reply:
I think spelling it out is the preferred option for me, I originally was following notation on the VPN SaaSboard but I think that is an acceptable difference.